### PR TITLE
Use qt5keychain when building with Qt 5.

### PR DESCRIPTION
**Pull request by rryan**
Also defaults secure credentials on if building with QtKeychain.

### CODE REVIEW

File: build/depends.py [hunk 1]
```diff
@@ -692,10 +692,9 @@ def sources(self, build):
 
 class QtKeychain(Dependence):
     def configure(self, build, conf):
-        libs = ['qtkeychain']
-        if not conf.CheckLib(libs):
-            raise Exception(
-                "Could not find qtkeychain.")
+        lib = 'qt5keychain' if Qt.qt5_enabled(build) else 'qtkeychain'
+        if not conf.CheckLib(lib):
+            raise Exception("Could not find qtkeychain.")
```
Comment (daschuer) [inline build/depends.py]: this can be misleading if qtkeychain is installed, but qt5keychain is missing.

**Comment by daschuer** [comment]
Thank you for the fix. I think this bugfix is useful in the 2.1 branch as well. 
Can you rebase it to 2.1?

**Comment by rryan** [comment]
If Qt5 is enabled then I think qt5keychain must be used - so I don't think
there's a way around it? What do you suggest I do instead?

**Comment by daschuer** [comment]
The PR is fine, we just need to report that qt5 version is missing in the qt5 case. 

And it would be nice to have this fix in the 2.1 branch as well.

**Comment by daschuer** [comment]
LGTM, Thank you.

**Pull request merged**